test/Closures/rlexe.xml

```diff
@@ -158,4 +158,10 @@
       <tags>BugFix</tags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>bug_OS_9008744.js</files>
+      <tags>BugFix,exclude_dynapogo</tags>
```

Comment:
We could I suppose.  The parsing logic that was causing the bug is now independent of buildAST flag though.  What do you think?